documentation/library_guide/api_for_sycl_kernels/tested_standard_cpp_api.rst

```diff
@@ -62,397 +62,802 @@ The printed result is:
 Tested Standard C++ API Reference
```

Comment:
Up to me - the old style of this table is more readable, so I prefer the old style.
I think that ability to read "as table" is more important in this file than then difficulties to edit it.

Reply:
@ValentinaKats what do you think? It sounds like we have more votes for the old style.

Reply:
I prefer new style more because it's less error-prone. With old style any extra "*" symbol leads to the table not being generated and usually it takes us time to notice it. 
Is there a way to enable some kind of syntax checks for rst pages?

Reply:
In this very table, I do not see a single "*" symbol :)

The very first column can be extended for more space, it that helps. But overall, thanks to the aligned dashed lines between the table rows, it's easy to observe that a new or edited row does not fit some of the column size.

Do not get me wrong, I am not saying that each table should be reformatted to use the simple style. But my preference is to keep it looking like a table, if it is simple enough for that. This exact table is simple.

Also note that it can be prepended with the `.. table::` directive that allows setting e.g. relative column width.

Reply:
>... leads to the table not being generated and usually it takes us time to notice it.

I thought our CI has a job do generate the documentation from RSTs, which should fail if there are errors. Do I miss something?

Reply:
When I build my docs, I check for errors. There were none when I published to IDZ (the intel website), so a change was introduced after I published for the 2022.0 release. I don't mind reverting back to the old table, the new version was suggested by a coworker after I spent a bunch of time trying to find/fix/rebuild the docs. 

Let's close this PR with no changes and revisit changing the tables later if we need to.